python/lsst/ts/watcher/alarm.py

```diff
@@ -36,6 +39,32 @@ class Alarm:
         Name of alarm. This must be unique among all alarms
         and should be of the form system.[subsystem....]_name
         so that groups of related alarms can be acknowledged.
+
+    Attributes
+    ----------
+    name : str
```

Comment:
A friendly suggestion to enclose python types in back ticks here and further down as well.

Reply:
Good point. I have gone through and looked for all instances where the back-ticks were missing. I fixed all the instances with a specific type (such as str and bool) and left the squishy ones alone (such as callable and DDS data) since Sphinx cannot usefully link to those.

This package doesn't yet use mypy and I'm a bit reluctant to bring the typing module into the mix yet.